src/models/user.js

```diff
@@ -4,9 +4,17 @@ export default {
   state: {
     favoriteControllers: [],
     favoriteResults: [],
+    user: {},
   },
 
   effects: {
+    *loadUser({ payload }, { put }) {
+      console.log(payload);
```

Comment:
Our linter should have caught this console.log() statement here. Do we need this here?

Reply:
solved!